packages/server/src/fhir/operations/utils/parameters.ts

```diff
@@ -233,6 +233,16 @@ export function buildOutputParameters(operation: OperationDefinition, output: ob
 }
 
 function makeParameter(param: OperationDefinitionParameter, value: unknown): ParametersParameter | undefined {
+  if (param.part && value && Array.isArray(value)) {
+    const parts: ParametersParameter[] = [];
+    for (const val of value.map((v) => makeParameter(param, v))) {
+      if (val) {
+        parts.push(val);
+      }
+    }
+    return { name: param.name, part: parts };
+  }
```

Comment:
Would you mind adding a specific unit test case for this in `parameters.test.ts`?